### PR TITLE
design static pages

### PR DESCRIPTION
**Pull request by kieutrong**

### CODE REVIEW

**Comment by kieutrong** [comment]
ready

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          </div>
+          <div class="col-md-2">
+            <img src="assets/pay.jpg" alt="" />
+          </div>
+        </div>
+      </div>
+      <div class="lh col-md-2 col-md-offset-2">
+        <div class=" row">
+          <div class="col-md-3">
+            <%= link_to image_tag("face.png", :alt => "facebook", :title => "face",:class => "via-img"), "#" %>
```
Comment (huongnguyenmta) [inline app/views/layouts/_footer.html.erb]: - `<%= link_to image_tag("face.png", alt: "facebook", title: "face", class : "via-img"), "#" %>`
- I18n `facebook`, `title`

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+          </div>
+          <div class="col-md-3 ">
+            <%= link_to image_tag("youtube.png", :alt => "youtube", :title => "youtube",:class => "via-img"), "#" %>
```
Comment (huongnguyenmta) [inline app/views/layouts/_footer.html.erb]: Tương tự như trên

File: app/views/layouts/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+          </div>
+          <div class="col-md-3 ">
+            <%= link_to image_tag("google.png", :alt => "google", :title => "google",:class => "via-img"), "#" %>
```
Comment (huongnguyenmta) [inline app/views/layouts/_footer.html.erb]: Tương tự

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     <%= csrf_meta_tags %>
-
     <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
   </head>
-
   <body>
-    <%= yield %>
+    <%= render "layouts/header" %>
+      <%= yield %>
```
Comment (huongnguyenmta) [inline app/views/layouts/application.html.erb]: indent

File: spec/controllers/static_pages_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/controllers/static_pages_controller_spec.rb]: using double quote

File: spec/models/comment_spec.rb [hunk 6]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/comment_spec.rb]: using double quote

File: spec/models/post_spec.rb [hunk 7]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/post_spec.rb]: using double quote

File: spec/models/post_tag_spec.rb [hunk 8]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/post_tag_spec.rb]: using double quote

File: spec/models/relationship_spec.rb [hunk 9]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/relationship_spec.rb]: using double quote

File: spec/models/tag_spec.rb [hunk 10]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/tag_spec.rb]: using double quote

File: spec/models/user_spec.rb [hunk 11]
```diff
@@ -1,5 +1,4 @@
 require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/models/user_spec.rb]: using double quote

File: spec/rails_helper.rb [hunk 12]
```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+require 'spec_helper'
```
Comment (huongnguyenmta) [inline spec/rails_helper.rb]: using double quote

File: spec/rails_helper.rb [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+ENV['RAILS_ENV'] ||= 'test'
```
Comment (huongnguyenmta) [inline spec/rails_helper.rb]: using double quote

File: spec/rails_helper.rb [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+require File.expand_path('../../config/environment', __FILE__)
```
Comment (huongnguyenmta) [inline spec/rails_helper.rb]: using double quote

File: spec/rails_helper.rb [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'rspec/rails'
```
Comment (huongnguyenmta) [inline spec/rails_helper.rb]: using double quote

File: spec/views/static_pages/help.html.erb_spec.rb [hunk 16]
```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/views/static_pages/help.html.erb_spec.rb]: using double quote

File: spec/views/static_pages/introduce.html.erb_spec.rb [hunk 17]
```diff
@@ -0,0 +1,4 @@
+require 'rails_helper'
```
Comment (huongnguyenmta) [inline spec/views/static_pages/introduce.html.erb_spec.rb]: using double quote

**Comment by huongnguyenmta** [comment]
commented

**Comment by kieutrong** [comment]
ready

**Pull request merged**